src/annotator/integrations/test/pdf-metadata-test.js

```diff
@@ -85,15 +104,17 @@ class FakePDFViewerApplication {
       eventBusEvents = true,
       initializedPromise = true,
       newFingerprintAPI = true,
+      withDownloadComplete = true,
     } = {},
   ) {
     this.url = url;
     this.documentInfo = undefined;
     this.metadata = undefined;
-    this.pdfDocument = null;
+    this.pdfDocument = new FakePDFDocumentProxy();
```

Comment:
We need this to be set from the beginning now, so I changed it so that it can be created "empty", but exposes a `finishLoading` method that is invoked the the fake app's own `finishLoading` method is called.

Reply:
Is it possible in PDF.js that `pdfDocument` is unset either temporarily or permanently when Hypothesis is loaded? Do we handle that gracefully?

Reply:
According to our type definitions, it is mandatory, but looking at the code, it can be null apparently https://github.com/mozilla/pdf.js/blob/f68310b7b1db39a934124acc90867a949da2d091/web/app.js#L971

Also, if you check that code, the prop is initialized only when `load` method is called.

I'll create an issue to try to reproduce what happens when it is not set and handle it properly.